src/calendar/header/index.tsx

```diff
@@ -184,7 +184,7 @@ const CalendarHeader = forwardRef((props: CalendarHeaderProps, ref) => {
       }
 
       return (
-        <Text allowFontScaling={false} key={index} style={dayStyle} numberOfLines={1} accessibilityLabel={''}>
+        <Text allowFontScaling={false} key={index} style={[dayStyle, {flex: 1}]} numberOfLines={1} accessibilityLabel={''}>
```

Comment:
This is bad for performance

Reply:
The `dayStyle` defined the width to be 32, flexing them can result in miss-alignment as they need to be alight to the day numbers